microsite/docs/ConnectingData/AWS.md

```diff
@@ -9,7 +9,7 @@ Your AWS account needs to be configured to generate Cost and Usage reports and s
 
 1.  Ensure your aws account has the correct permissions
 
-    - You will need an [IAM](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/) user that can create access-keys and modify your billing settings.
+    - IAM permissions are not required to run the app.
```

Comment:
What we want to convey here is that you do not need to create a new IAM user or access keys, you just need an IAM role that will give you access to to Billing data via the CUR and Athena.

I think it is still fine to remove this link, but we may want to change the documentation to reflect the key point here.

Reply:
Hi @camcash17 
Please check now